python/lsst/ip/isr/overscan.py

```diff
@@ -79,6 +80,12 @@ class OverscanCorrectionTaskConfig(pexConfig.Config):
         doc="Correct using parallel overscan after serial overscan correction?",
         default=False,
     )
+    maskPixFracThresh = pexConfig.Field(
```

Comment:
This should be a `RangeField` which allows you to specify a min/max and whether they are inclusive or not (they are in this case), and then that handles the checks behind the scenes.

Reply:
I don't like this name because it doesn't tell us that this is about parallel overscan.